packages/notification-center/src/components/notification-center-web-component/notification-center-component.tsx

```diff
@@ -110,6 +111,7 @@ function PopoverWrapper({
   popoverConfig,
   unseenBadgeColor,
   unseenBadgeBackgroundColor,
+  preferenceFilter,
```

Comment:
This should also be passed to the NotificationCenterComponent props above (line 68)